sequelizeDatabase/sequelFunctions.js

```diff
@@ -23,35 +29,85 @@ getAllEmployeeHistory = () => {
         return employeeHistory;
     }).catch(err => console.error(err));
 };
-getAllFundCompany = () => {
-    return models.FundCompany.findAll().then((fundCompany) => {
-        return fundCompany;
+
+getAllOriginalFundPositions = () => {
+    return models.OriginalFundPosition.findAll().then((originalPositions) => {
+        return originalPositions;
     }).catch(err => console.error(err));
-};
+}
+
+getAllSharedFunds = () => {
+    return models.SharedFunds.findAll().then((sharedFunds) => {
+        return sharedFunds;
+    }).catch(err => console.error(err));
+}
 
 //--------------------Insert into Table Functions-----------//
 
-insertPerson = (IndividualName, OriginalPosition, LinkedInUrl, Comments) => {
-    return models.Individuals.findOrCreate({where:{IndividualName: IndividualName}, 
+insertUser = (Username, Password, Email) =>{
+    return models.User.findOrCreate({where:{Email: Email},
+        defaults: {
+            Username: Username,
+            Password: Password
+        }
+    }).spread((user, createdBoolean) =>{
+        return user;
+    }).catch(err => console.error('Error in "insertUser", ', err));
+};
+
+insertFund = (FundName, UserID) => {
+    return models.Funds.create({ //TODO: Make this a findOrCreate to prevent duplicate names.
+        FundName: FundName,
+        UserID: UserID
+    }).then((createdFund) => {
+        return createdFund;
+    }).catch(err => console.error('Error in "insertFund", ', err));
+};
+
+insertSharedFunds = (FundID, UserID) => {
+    //NOTE: UserID here is the user the fund is being shared with, not the fund owner.
```

Comment:
Going with 376's clean code recommendations, you should probably name the variable to indicate and not leave comments there.